docs/user/filesystem.md

```diff
@@ -1021,6 +1021,12 @@ libpil4dfs intercepting summary for ops on DFS:
 [op_sum ]  5003
 ```
 
+### Bypassing function interception in libpil4dfs
+libpil4dfs could boost IO performance considerably over using dfuse for IO intensive workloads. However, initializing daos environment in libpil4dfs is expensive. To alleviate the overhead of libpil4dfs, users could set env to disable function interception in specific applications. I.e., let dfuse handle all IO requests. Users can set env "D_IL_BYPASS_LIST" to bypass function interceptions in target processes and their child processes. This is recommended for short-lived processes (shorter than 100 milliseconds) that run a large number (more than 1,000) of times. Example to set env:
```

Comment:
Do you really want to disable for child processes?   Just thinking you may get a situation where client runs their application via a shell script and they want to ignore everything but the application.   In fact, it might be better to have a D_IL_INTERCEPT_LIST (which by default is "all")

Reply:
If interception is turned on only for applications provided by D_IL_INTERCEPT_LIST, users need to have enough knowledge about their workflow. Some users may just take scripts from their colleagues and they are not very familiar which applications have intensive IO. Currently interception is default on for applications except those in "D_IL_BYPASS_LIST" and built-in bypass list.

Reply:
Considering cases like "configure", disabling interception in all child processes would be easier to set up. Otherwise, we need to add all applications involved into the bypass list.

Reply:
@jolivier23 I updated this PR to bypass the interception in individual process only. Children processes are not affected now. Thank you!